lstchain/scripts/lstchain_dl2_to_dl3.py

```diff
@@ -109,7 +109,7 @@ def main():
     # Assuming the filename to be 'dl2_LST-1.Run#####_*.h5'
```

Comment:
There are functions for this in lstchain:

```
from lstchain.paths import run_info_from_filename
run_info = run_info_from_filename(filename)
```

Reply:
Thanks for pointing this out!